example/test/function/index.js

```diff
@@ -0,0 +1,11 @@
+const fastify = require('fastify')
```

Comment:
can you add python example?

Reply:
I added the python sample as well. Also added the instructions for both samples to the README.md.